Armstrong_Number/Armstrong_Number.rb

```diff
@@ -0,0 +1,54 @@
+#A positive integer of n digits is called an Armstrong number of order n (order is number of digits) if
+#abcd... = pow(a,n) + pow(b,n) + pow(c,n) + pow(d,n) + .... 
+
+#To calculate the order of the number
+def order(num)
+	# variable to store of the number 
```

Comment:
4 space indentation is not followed.

Reply:
Have I need indent the whole code with comments 4 indent or only this part.